vvp-dossier/example-vvp-dossier.json

```diff
@@ -0,0 +1,28 @@
+{
+  "v": "ACDC10JSON0002fb_",
+  "d": "ENKvBDgVxcoLY_KghDq5IoVPyOz2MQGwSZ0yq6aD4-4_",
+  "i": "EAt3TsAudsZWhQ-U8OPaBZqBaLWqpCMNWsDRPG3-UBWd",
+  "ri": "EH8y2GjzeBonizRk6oqAyC9ER5OSFiQXOObGJKTA8Ldr",
+  "s": "EL3mANFohCJSfsZVc3jrqIKEZsLC-KdCC455UCOGPWGi",
+  "a": {
+    "d": "EFZkbTkgS3jPpc1BOD_KusEM8p7VDzycVZiw2ou-tLzK",
+    "dt": "2024-12-17T13:17:48.296000+00:00",
+    "LEI": "549300QME1XNVC6ESHEL"
+  },
+  "e": {
```

Comment:
There needs to be an edge to the vetting cred.

We know we will need an edge to the brand and brand proxy cred after January; why not add them as optional edges now?

Reply:
added vetting as edge and also added brand owner and brand proxy as edge